examples/todoapp/lib/feature_todo_list/views/todo_list_page.dart

```diff
@@ -25,7 +24,7 @@ class TodoListPage extends StatelessWidget {
           title: Text(context.l10n.todos),
           actions: const [
             TodosFilterPopupMenuButton(),
-            AppTodoListBulkEditPopupMenuButtonWithDependencies(),
+            AppTodoListBulkEditPopupMenuButton(),
```

Comment:
Don't we need the dependencies here ?

Reply:
It's same as above. Using the `*WithDependencies` makes testing impossible, because this way you cannot provide the mock dependencies.

Reply:
Please use the approach described here
https://github.com/Prime-Holding/rx_bloc/issues/700